nextflow_schema.json

```diff
@@ -757,23 +757,18 @@
                     "description": "Specify which tool for bin quality-control validation to use.",
                     "enum": ["busco", "checkm"]
                 },
-                "busco_reference": {
+                "busco_db": {
                     "type": "string",
-                    "description": "Download path for BUSCO lineage dataset, instead of using automated lineage selection.",
-                    "help_text": "E.g. https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2020-03-06.tar.gz. Available databases are listed here: https://busco-data.ezlab.org/v5/data/lineages/."
-                },
-                "busco_download_path": {
-                    "type": "string",
-                    "description": "Path to local folder containing already downloaded and unpacked lineage datasets.",
-                    "help_text": "If provided, BUSCO analysis will be run in offline mode. Data can be downloaded from https://busco-data.ezlab.org/v5/data/ (files still need to be unpacked manually). Run in combination with automated lineage selection."
+                    "description": "Download path for BUSCO lineage dataset or path to local directory containing already downloaded and unpacked lineage datasets.",
+                    "help_text": "E.g. https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2020-03-06.tar.gz or '/path/to/buscodb' (files still need to be unpacked manually). Available databases are listed here: https://busco-data.ezlab.org/v5/data/lineages/."
                 },
                 "busco_auto_lineage_prok": {
                     "type": "boolean",
                     "description": "Run BUSCO with automated lineage selection, but ignoring eukaryotes (saves runtime)."
                 },
-                "save_busco_reference": {
+                "save_busco_db": {
```

Comment:
I'm wondering if this should keep its name (or if it should be renamed to "save_busco_references"? As far as I see it, the use case for this parameter is to save the lineages files downloaded when using online auto lineage detection. On their own, these don't actually comprise a busco database as there are other index files you need to download in order to pass a directory to `busco_db`.